texastribune/donations
# Stop sending `openended_status` and `installments`

#### What's this PR do?
Removes the `openended_status` and `installments` form values from front and back end.

#### Why are we doing this? How does it help us?
We don't need them now that Circle, BusMem and regular donations have discreet pages.

#### How should this be manually tested?
`make`, `make restart`, then make one time, monthly and yearly donations on `/circle`, `/business` and `/circle`. Confirm the proper amount and frequency show up in #stripe.

#### How should this change be communicated to end users?
NA.

#### Are there any smells or added technical debt to note?
NA.

#### What are the relevant tickets?
https://3.basecamp.com/3098728/buckets/10357222/todos/1787329290

#### Have you done the following, if applicable:
